dockerfiles/typescript.Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Build in a full featured container
 ARG TARGETARCH
-FROM --platform=linux/$TARGETARCH node:20-bullseye AS build
+FROM --platform=linux/$TARGETARCH node:20-bookworm AS build
```

Comment:
Newest Debian release. I was seeing some errors and this seemed to have fixed it.